Source/Tanikaze/tanikaze.h

```diff
@@ -422,7 +422,7 @@ KDU_DB_ENTRY gProvEntry[] = {
 extern "C" {
 #endif
 
-    KDU_DB gProvTable = {
+    __declspec(dllexport) KDU_DB gProvTable = {
```

Comment:
This export is defined in module definition file, "export.def". This explicit declaration in this case will cause linker warning: 
>LNK4197: export 'gProvTable' specified multiple times; using first specification